modules/administration/pages/auth-methods-pam.adoc

```diff
@@ -0,0 +1,52 @@
+[[auth-methods]]
+= Authentication With PAM
+
+{productname} supports network-based authentication systems using pluggable authentication modules (PAM).
+PAM is a suite of libraries that allows you to integrate {productname} with a centralized authentication mechanism, eliminating the need to remember multiple passwords.
+{productname} supports LDAP, Kerberos, and other network-based authentication systems using PAM.
+
+
+
+.Procedure: Enabling PAM
+
+. Create a PAM service file at [path]``/etc/pam.d/susemanager``.
+A standard [path]``/etc/pam.d/susemanager`` file should look like this.
+It configures {productname} to use the system wide PAM configuration:
++
+----
+#%PAM-1.0
+auth     include        common-auth
+account  include        common-account
+password include        common-password
+session  include        common-session
+----
+. Enforce the use of the service file by adding this line to [path]``/etc/rhn/rhn.conf``:
++
+----
+pam_auth_service = susemanager
+----
++
+In this example, the PAM service file is called [systemitem]``susemanager``.
+. Restart the {productname} services after a configuration change.
+. In the {productname} {webui}, navigate to [menuitem]``Create User`` and enable a new or existing user to authenticate with PAM.
```

Comment:
```suggestion
. In the {productname} {webui}, navigate to menu:Users[Create User] and enable a new or existing user to authenticate with PAM.
```
"Create User" is probably wrong if you want to edit an existing user.  This probably must be done with the user's details page.  See reference:pages:users/user-details.adoc.  Hope I got it right.